pkg/controllers/componentroutestatus/ingress_component_route.go

```diff
@@ -0,0 +1,89 @@
+package componentroutestatus
```

Comment:
@stlaz asked me offline to merge this into component_route_secret_sync_controller.

Reply:
Done.